docs/README.md

```diff
@@ -1,154 +1,233 @@
 
-## Style guide
+2. Clone the repository to your local machine.
 
-If you have questions about how to write for Docker's documentation (e.g.,
-questions about grammar, syntax, formatting, styling, language, or tone) please
-see the [style guide](sources/contributing/docs_style-guide.md). If something
-isn't clear in the guide, please submit a PR to help us improve it.
+3. Select an issue from `docker/docker` to work on or submit a proposal of your own.
 
-## Working using GitHub's file editor
+4. Create a feature branch from `master` in which to work.
 
-Alternatively, for small changes and typos you might want to use GitHub's built-
-in file editor. It allows you to preview your changes right on-line (though
-there can be some differences between GitHub Markdown and [MkDocs
-Markdown](http://www.mkdocs.org/user-guide/writing-your-docs/)).  Just be
-careful not to create many commits. And you must still [sign your
-work!](../CONTRIBUTING.md#sign-your-work)
+	By basing from `master` your work is automatically included in the next
+	release. It alos allows docs maintainers to easily cherry-pick your changes
+	into the `docs` release branch. 
 
-## Branches
+4. Modify existing or add new `.md` files to the `docs/sources` folder.
 
-| Branch   | Description                    | URL (published via commit-hook)                                              |
-|----------|--------------------------------|------------------------------------------------------------------------------|
-| `docs`   | Official release documentation | [http://docs.docker.com](http://docs.docker.com)                             |
-| `master` | Unreleased development work    | [http://docs.master.dockerproject.com](http://docs.master.dockerproject.com) |
+	If you add a new document (`.md`) file, you must also add it to the
+	appropriate section of the `docs/mkdocs.yml` file in this repository.
+	
+
+5.  As you work, build the documentation site locally to see your changes.
+
+	The `docker/docker` repository contains a `Dockerfile` and a `Makefile`.
+	Together, these create a development environment in which you can build and
+	run a container running the Docker documentation website. To build the
+	documentation site, enter `make docs` at the root of your `docker/docker`
+	fork:
+	
+		$ make docs
+		.... (lots of output) ....
+		docker run --rm -it  -e AWS_S3_BUCKET -p 8000:8000 "docker-docs:master" mkdocs serve
+		Running at: http://0.0.0.0:8000/
+		Live reload enabled.
+		Hold ctrl+c to quit.
+	
+	
+	The build creates an image containing all the required tools, adds the local
+	`docs/` folder and generates the HTML files. Then, runs a Docker
+	container with this image. 
+	
+	The container exposes port 8000 on the localhost so that you can connect and
+	see your changes. If you are running Boot2Docker, use the `boot2docker ip`
+	to get the address of your server.
+
+6.  Check your grammar and spelling against an online checker such as (http://www.hemingwayapp.com/). 
```

Comment:
We need to mention the style guide here (and maybe elsewhere). It supersedes anything a grammar checker might suggest. They need to know about it in general, anyway.